### ISSUE TITLE
Minified Exception

### ISSUE THREAD

**Issue by jhoffmcd** [react-grid-layout/react-draggable#161]
Today I received this error when minifying my bundled JS files:

```
production.min.js:6 Uncaught Error: Minified exception occurred; use the non-minified dev environment for the full error message and additional helpful warnings.
```

Switching `NODE_ENV` to `development` yielded this message:

```
production.min.js:6 Uncaught Invariant Violation: Element appears to be neither ReactComponent nor DOMNode (keys: getBoundPosition,snapToGrid,canDragX,canDragY,getControlPosition,createCoreData,createDraggableData)
```

I tested both the `Draggable` and `DraggableCore` and I got the same message. I'm using React v15.0.1 and Draggable v2.1.0.

When non-minified, I'm getting expected behavior and the code works great.

Usage:

``` jsx
<DraggableCore onDrag={this.dragHandler}>
    <div className="c-spread__drag-handle js-drag-handle">               
    </div>
</DraggableCore>
```

**Comment by NickStefan** [comment]
Same issue here. It appears to possibly be a Flow --> ES6 conversion, issue?

In the flow es6 code:
line 72, https://github.com/mzabriskie/react-draggable/blob/master/lib/utils/positionFns.es6

```
// Create an data object exposed by <DraggableCore>'s events
export function createCoreData(draggable: DraggableCore, x: number, y: number): DraggableData {
  // ... ... ...
  if (isStart) {
    // If this is our first move, use the x and y as last coords.
    return {
      node: ReactDOM.findDOMNode(draggable),
      deltaX: 0, deltaY: 0,
      lastX: x, lastY: y,
      x: x, y: y
    };
  }
```

Becomes this after the es6 build:

```
function createCoreData(draggable, x, y) {
  // ... ... ...
  if (isStart) {
    // If this is our first move, use the x and y as last coords.
    return {
      node: _reactDom2.default.findDOMNode(this),
      deltaX: 0, deltaY: 0,
      lastX: x, lastY: y,
      x: x, y: y
    };
  }
```

`THIS !== draggable`, in ReactDOM.find, in minifeid builds, and thus creates problems. I believe this is the issue. Any idea on how to solve this?

**Comment by NickStefan** [comment]
I see this was actually fixed in v2.1.1 @jhoffmcd , like 2 days ago. Will upgrade package.

**Comment by jhoffmcd** [comment]
Confirmed working with v2.1.1. Thanks for the heads up @NickStefan !
